gateways/adyen/request_builder.go

```diff
@@ -130,3 +150,9 @@ func buildVoidRequest(voidRequest *sleet.VoidRequest, merchantAccount string) *p
 	}
 	return request
 }
+
+func addIfNonEmpty(value string, key string, data *map[string]string) {
```

Comment:
@niraj this seemed too specific to put in utils but what do you think?

Reply:
Yeah lets keep it here for now but if we find ourselves using something like this again we can revisit?